### ISSUE TITLE
Storybook Provider addon: Express input:checkbox is missing id to be labelled by htmlFor prop

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4067]
https://github.com/adobe/react-spectrum/blob/e9f08177302590b4bf8c650dda579a15f0c5ffcb/.storybook/custom-addons/provider/register.js#L113-L114

should be

```tsx
 <label htmlFor="express">Express: </label> 
 <input type="checkbox" id="express" name="express" onChange={onExpressChange} checked={values.express} /> 
```
